### DISCUSSION TITLE
My account restricted

### DISCUSSION THREAD

**Discussion by VENDETTA2212** [codewars/codewars.com#3122] [Q&A] [answered]
My account is restricted and I cheated, but not that much. I got kyu 3 without any copying or cheating or any weird solution. but after kyu 3 I copied 4 kata (3 x kyu1 and 1 x kyu2) solutions from the internet, and I agree that I cheated, but after my account got restricted they considered my other solutions cheating and I did not get the points for them too. Can someone check and at least give me my points for katas that I solved myself without any cheating? ( I just cheated on the last 4 katas in my profile )
My account name is "VENDETTA2212"

**Comment by Blind4Basics** [comment]
Hello,

Outside of those 4, only the blue kata got invalidated, and they were because most of them were also copies. All other levels are untouched. 
Courses of actions:
1. I can revalidate some blue (3-4, no more) that could be valid and unlock the account
2. you can delete it and start fresh.

Option 1, I guess?

**Reply by VENDETTA2212** [reply]
About kyu 4 katas, is there a way for me to prove that I've done them myself ?(they are all at most completed list of kyu 4, if I wanted to cheat, I would go for least completed ones too) I didn't cheat on kyu 4 katas, maybe my code was similar to someone else but I didn't cheat (why should I cheat on kyu 4 when I could cheat on kyu 1). Please delete those 4 kyu 1 katas and any kyu 3 that you think I've cheated on (because they are not many and I can solve them again). I have been working on 2 katas to publish but now I don't have access to complete them, please fix that too, I will never copy anything from anywhere for any kata again and I will also tell my friends about this so that they don't do this. you have my word

**Comment by Blind4Basics** [comment, marked as answer]
> About kyu 4 katas, is there a way for me to prove that I've done them myself ?

No. Consider this as a penalty for trying to cheat.

> (they are all at most completed list of kyu 4, if I wanted to cheat, I would go for least completed ones too). I didn't cheat on kyu 4 katas, maybe my code was similar to someone else but I didn't cheat (why should I cheat on kyu 4 when I could cheat on kyu 1).

I'm not sure I understand this sentence, sorry. About what ppl would do or not when it comes to cheating, experience proved that anything can happen. ;)

> Please delete those 4 kyu 1 katas and any kyu 3 that you think I've cheated on (because they are not many and I can solve them again).

Note:
* you didn't complete any 3 kyu
* when invalidated a kata cannot grant reward anymore (honor/progress)

---

Your account is now unlocked (hence you can publish kata again), and the 4 kyu that seemed potentially legit got revalidated.

Cheers

**Reply by VENDETTA2212** [reply]
But my kyu 4 katas were not copies, I have been working for rank of kyu 3 since middle of spring, take a look at the submit time of katas, I did not solve them in a short period of time. Just tell me how can I prove that my 3 kyu rank was honorable and without cheating

**Reply by Blind4Basics** [reply]
Look, you're currently at 75% in 4 kyu level. That's just one 2kyu kata away from you being 3 kyu again. So just pick an easy one like the assembler interpreter II and you'll be there again (solving it in a legit way, ofc).
You cheated, you got caught. so just get on with it. Keep in mind the penalty stays very low, in your case.

Cheers

**Reply by VENDETTA2212** [reply]
If you don't want to give them back to me, at least give me the chance to solve them again because I can solve them and I didn't cheat on them. I cheated at kyu 3, developed to kyu 2, got caught, and now I'm in kyu 4, is that fair? I'm just asking for another chance to solve them again... No more

**Reply by VENDETTA2212** [reply]
By the way if I could solve kyu2 katas, I wouldn't copy solutions for them.

**Reply by Blind4Basics** [reply]
No, sorry.

> If you don't want to give them back to me, at least give me the chance to solve them again 

Also, those two ideas are technically incompatible.

Cheers

edit:

> By the way if I could solve kyu2 katas, I wouldn't copy solutions for them.

Just train harder, then? or again, pick a candidate that suites you better. Just actually work your way there, you'll end up solving those too (legit).

**Reply by VENDETTA2212** [reply]
I'm insisting for getting them back because I worked really hard for kyu 4 katas and they were the best I could do without cheating. I agree, I cheated, I am sorry and will never do that again.
